test/internet/test-dns-ipv6.js

```diff
@@ -134,62 +133,64 @@ TEST(function test_lookup_ipv6_hint(done) {
         throw err;
```

Comment:
I would  do `return done();` as in other places in the file and would get rid of the `else`